src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 50;
+  const alternativeDicount = 20;
+  const basicCost = 40;
+
+  let sum = 0;
+
+  for (let i = 1; i <= days; i++) {
+    sum += basicCost;
+  }
```

Comment:
mb you can just multiplication `days` on `basicCost` without `for`

Reply:
Agree, check @vladyslav-lapshyn comment, we can avoid using loops here